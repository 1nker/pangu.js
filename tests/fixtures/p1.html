<div id="p1">
    <h1>好abc個秋</h1>
    <div>
        <p><a href="#">abc1</a>漢字一</p>
        <p><a href="#">abc2</a><a href="#">漢字二</a></p>
    </div>
    <div>
        <p>123漢字abc</p>
        <p>漢字abc</p>
        <br>
        <p>漢字123</p>
    </div>
    <div>
        <p>123漢字abc</p>
        <p>123漢字<br>abc</p>
        <br>
        <textarea>中文abc漢字5566</textarea>
        <p>中文7788</p>
        <textarea><div>中文abc漢字5566<span>漢字123</span></div></textarea>
        <label>選項(必填) ex</label>
        <input type="text" value="中文123漢字abc" />
        <p>中文2266</p>
    </div>
    <div>
        <div contenteditable="true">abc漢字1</div>
        <div contenteditable="false">abc漢字2</div>
        <div g_editable="true">abc漢字3</div>
        <div g_editable="false">abc漢字4</div>
    </div>
</div>
