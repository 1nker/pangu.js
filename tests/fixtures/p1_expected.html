<div id="p1">
    <h1>好 abc 個秋</h1>
    <div>
        <p><a href="#">abc1</a> 漢字一</p>
        <p><a href="#">abc2</a><pangu> </pangu><a href="#">漢字二</a></p>
    </div>
    <div>
        <p>123 漢字 abc</p>
        <p>漢字 abc</p>
        <br>
        <p>漢字 123</p>
    </div>
    <div>
        <p>123 漢字 abc</p>
        <p>123 漢字<br>abc</p>
        <br>
        <textarea>中文abc漢字5566</textarea>
        <p>中文 7788</p>
        <textarea><div>中文abc漢字5566<span>漢字123</span></div></textarea>
        <label>選項 (必填) ex</label>
        <input type="text" value="中文123漢字abc" />
        <p>中文 2266</p>
    </div>
    <div>
        <div contenteditable="true">abc漢字1</div>
        <div contenteditable="false">abc 漢字 2</div>
        <div g_editable="true">abc漢字3</div>
        <div g_editable="false">abc 漢字 4</div>
    </div>
</div>
